README.md

```diff
@@ -41,16 +41,18 @@ Additionally, the library offers an interface to encode SVG images.
 ## Prerequisites
 
 ```
-sudo apt install cpp cmake librsvg2-dev libx264-dev libgtk-3-dev
+sudo apt install gcc-c++ #Fedora
+sudo apt install g++ #Ubuntu
+sudo apt install cmake librsvg2-dev libx264-dev libgtk-3-dev
+sudo apt-get install libavcodec-dev libavfilter-dev libavformat-dev libavutil-dev
 ```
 
 ## Building
 
 ```
-git clone https://github.com/apc-llc/moviemaker-cpp.git
+git clone --recurse-submodules -j8 https://github.com/apc-llc/moviemaker-cpp.git
 cd moviemaker-cpp
-mkdir build
-cd build
+mkdir build && cd $_
```

Comment:
Let's not confuse the user with bash tricks. `$_` is a cool feature, but it is far less obvious than `cd build`. The point of having the build instructions is not just to copy-paste them. The user should be able to actually *understand* what has to be done, and creatively adapt for his own environment as needed. It's like in a C program you may want to intentionally create a temporary var `bool createBuildFolder = true` instead of just passing `true` to a function, because it's very helpful for understanding.

Reply:
@dmikushin I agree with you. Can you merge now?